requirements.txt

```diff
@@ -1 +1,3 @@
 ckantoolkit
+# remove when errors with "packaging" package is fixed
+setuptools==65.3.0
```

Comment:
This will 100% cause  problems down the line, extensions should not touch setuptools. What errors is this trying to fix?

Reply:
The error in the _Install requirements_ test step was

<pre>
Traceback (most recent call last):
  File "/__w/ckanext-announcements/ckanext-announcements/setup.py", line 2, in <module>
    from setuptools import setup, find_packages
  File "/usr/lib/python3.10/site-packages/setuptools/__init__.py", line 242, in <module>
    monkey.patch_all()
  File "/usr/lib/python3.10/site-packages/setuptools/monkey.py", line 99, in patch_all
    patch_for_msvc_specialized_compiler()
  File "/usr/lib/python3.10/site-packages/setuptools/monkey.py", line [13](https://github.com/okfn/ckanext-announcements/actions/runs/5788357671/job/15687190753#step:6:14)6, in patch_for_msvc_specialized_compiler
    msvc = import_module('setuptools.msvc')
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/usr/lib/python3.10/site-packages/setuptools/msvc.py", line 32, in <module>
    from packaging.version import LegacyVersion
<b>ImportError: cannot import name 'LegacyVersion' from 'packaging.version' </b> (/usr/lib/python3.10/site-packages/packaging/version.py)
</pre>

@amercader What do you think of moving this fix to tests? https://github.com/okfn/ckanext-announcements/commit/dec537852943c1f15dd1e35343a9aaf809580ae6